rpcs3/Loader/PSF.cpp

```diff
@@ -186,8 +188,10 @@ namespace psf
 		return result;
 	}
 
-	void save_object(const fs::file& stream, const psf::registry& psf)
+	std::vector<u8> save_object(const psf::registry& psf)
 	{
+		fs::file stream = fs::make_stream<std::vector<u8>>();
```

Comment:
Also stream can be made from existing vector with reserved memory, for example.

Reply:
Added an optional vector arg.